packages/core-products/src/module/configureProductsModule.ts

```diff
@@ -191,15 +196,21 @@ export const configureProductsModule = async ({
       return Products.findOne(selector, {});
     },
 
-    findProducts: async ({ limit, offset, ...query }) => {
+    findProducts: async ({ limit, offset, queryString, ...query }) => {
       const options: FindOptions = { sort: { sequence: 1, published: -1 } };
+      const filter = { ...query };
       if (limit) {
         options.limit = limit;
       }
       if (offset) {
         options.skip = offset;
       }
-      const products = Products.find(buildFindSelector(query), options);
+      if (queryString) {
+        const productIds = await productTexts.searchTexts({ searchText: queryString });
```

Comment:
Remove this please, it adds weight, it's not consistent to search for product title, subtitle compared to all the other searches and it was not part of the list here: https://github.com/unchainedshop/unchained/issues/436